src/components/footer-wide/index.scss

```diff
@@ -0,0 +1,231 @@
+
+      @media (min-width: 1200px) {
+        margin-bottom: 50px;
+      }
+    }
+
+    &__logo {
+      width: 30px;
+      height: fit-content;
+    }
+
+    &__title {
+      font-size: 25px;
+      font-weight: 700;
+      color: var(--neutral-main-cards);
+    }
+
+    &__content {
+      line-height: 1.5;
+      font-size: 15px;
+      font-weight: 700;
+      color: var(--neutral-silver-text);
+      margin-bottom: 25px;
+      display: -webkit-box;
+      -webkit-line-clamp: 3;
+      -webkit-box-orient: vertical;
+      overflow: hidden;
+
+      @media (min-width: 120px) {
+        -webkit-line-clamp: 2;
+      }
+    }
+
+    &__data {
+      font-size: 14px;
+      font-weight: 700;
+      color: var(--neutral-silver-text);
+      margin-bottom: 14px;
+
+      &:last-child {
+        margin-bottom: 0;
+      }
+    }
+
+    &__icon {
+      width: 14px;
+      vertical-align: middle;
+    }
+  }
+
+  &__links {
+    width: 25%;
+    padding: 0 12px;
+
+    @media (min-width: 1200px) {
+      width: 16.666%;
+      order: 4;
+      padding-left: 37px;
+    }
+  }
+  &__about {
+    width: 25%;
+    padding: 0 12px;
+
+    @media (min-width: 1200px) {
+      width: 16.666%;
+      order: 2;
+      padding-left: 37px;
+    }
+  }
+  &__location {
+    width: 50%;
+    height: auto;
+    padding: 0 12px;
+
+    @media (min-width: 1200px) {
+      width: 25%;
+      order: 3;
+    }
+
+    &__map {
+      height: 200px;
+    }
+  }
+
+  &__topics {
+    width: 50%;
+    padding: 0 12px;
+
+    @media (min-width: 1200px) {
+      width: 25%;
+      order: 5;
+    }
+
+    &__item-box {
+      display: flex;
+      flex-direction: row;
+      gap: 25px;
+
+      &:last-child {
+        margin-top: 20px;
+      }
+
+      &__image-container {
+        width: 95px;
+        height: 95px;
+      }
+
+      &__image {
+        width: 100%;
+        height: 100%;
+        object-fit: cover;
+      }
+
+      &__info-container {
+        width: 65%;
+        display: flex;
+        flex-direction: column;
+        gap: 5px;
+
+        @media (min-width: 992px) {
+          width: 72%;
+        }
+
+        @media (min-width: 1200px) {
+          width: 54%;
+        }
+
+        @media (min-width: 1440px) {
+          width: 60%;
+        }
+      }
+
+      &__title {
+        font-size: 17px;
+        font-weight: 700;
+        color: var(--neutral-main-cards);
+
+        @media (min-width: 1440px) {
+          font-size: 19px;
+        }
+      }
+
+      &__content {
+        font-size: 14px;
+        font-weight: 700;
+        color: var(--neutral-silver-text);
+        display: -webkit-box;
+        -webkit-line-clamp: 3;
+        -webkit-box-orient: vertical;
+        overflow: hidden;
+      }
+
+      // &__image2 {
+      //   width: 280px;
+      //   height: 100px;
+      //   margin-bottom: 20px;
+
+      //   @media (min-width: 768px) {
+      //     width: 200px;
+      //     height: 150px;
+      //   }
+      // }
```

Comment:
Please, as I mentioned before: DO NOT preserve commented code, please, remove it and  DO NOT PUSH commented or dead code, thx.

Reply:
Resolved